source/administration/config-settings.rst

```diff
@@ -4295,6 +4295,23 @@ Experimental
 
 There are a number of settings considered "experimental" that are configurable from the System Console. These may be replaced or removed in a future release.
 
+Collapsed Reply Threads (Beta)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Collapsed Reply Threads offers users an enhanced user experience for organizing conversations with threads that are easier to find, follow, and resume. Collapsed Reply Threads are available from Mattermost v5.37, and are disabled by default. See our `Organizing Conversations using Collapsed Reply Threads (Beta) <https://docs.mattermost.com/help/messaging/organizing-conversations.html>`__ documentation to learn more about this feature.
+
+System Admins can set the default appearance of collapsed reply threads for their end users by going to **System Console > Experimental > Features**, then setting **Collapsed Reply Threads** to one of the following options:
+
+**Enabled (Default Off)**: Enable Collapsed Reply Threads functionality on the server. Users can choose to `enable Collapsed Reply Threads <https://docs.mattermost.com/help/settings/account-settings.html#collapsed-reply-threads-beta>`__ for their Mattermost account in **Account Settings**.
+
+**Enabled (Default On)**: Enable Collapsed Reply Threads functionality on the server, and enable it for all users. Individual users can disable this functionality in **Account Settings** for their Mattermost account, if preferred.
+
+**Disabled**: Disable Collapsed Reply Threads functionality.
+
++------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| This feature's ``config.json`` setting is ``"ServiceSettings.CollapsedThreads": disabled`` with options ``disabled``, ``default-on``, and ``default-off``.     |
```

Comment:
The order here after "...with options ..." should match the order that they are listed above